### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by hasertrene**
Review my code

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,77 @@
+/* Homepage */
```
Comment (FaabScheeren) [inline src/App.css:1]: Why did you put all the css only in one file?

File: src/components/Contact.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+export default function Contact() {
+  const today = new Date();
+  const time = today.getUTCHours()
+  const day = today.getUTCDay() > 1 ? today.getUTCDay() < 5 ? true : false : false
```
Comment (FaabScheeren) [inline src/components/Contact.js:6]: Looks nice!

File: src/App.css (hunk 1 above)
Comment (hasertrene) [reply src/App.css:1]: Yep. A mistake/lazy move.

File: src/pages/Database.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+
+const list = patients.filter((patient) => {
+  if (patient.doctorId === chosen.id) {
+  return true;
+} else if (chosen.id === 4){
+  return true;
+} else {
+  return false;
+}
```
Comment (stacey-miller) [inline src/pages/Database.js]: `const list = patients.filter(patient => patient.doctorId === chosen.id || chosen.id === 4)`

Comment (hasertrene) [reply src/pages/Database.js]: That makes me happy, thank you! :)

**Pull request merged**